libr/include/r_util/r_buf.h

```diff
@@ -101,6 +102,7 @@ R_API st64 r_buf_fwrite_at(RBuffer *b, ut64 addr, const ut8 *buf, const char *fm
 // entire buffer in memory. Consider using the r_buf_read* APIs instead and read
 // only the chunks you need.
 R_API const ut8 *r_buf_data(RBuffer *b, ut64 *size);
+R_API const ut8 *r_buf_data_cached(RBuffer *b, ut64 *size);
```

Comment:
remove this API, cache status should be just a "private" property and callers should not be aware of the fact that things are cached.